File: dataset/codenet/Java/Code/s476581255.java
import java.util.Scanner;
public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int[] a = new int[n];
    for(int i = 0; i<n; i++) {
      a[i] =sc.nextInt();
    }
    int count = bubbleSort(a, n);
    for(int i = 0; i<n; i++) {
      if(i<n-1){
        System.out.print(a[i] + " ");
      }else {
        System.out.println(a[i]);
      }
    }
    System.out.println(count);
  }

  public static int bubbleSort(int[] a,int n) {
    boolean flag = true;
    int count = 0;
    while(flag == true) {
      flag = false;
      for(int j = n-1;j>0; j--) {
        if(a[j]<a[j-1]) {
          int free = a[j];
          a[j] = a[j-1];
          a[j-1] = free;
          count++;
          flag = true;
        }
      }
    }
    return count;
  }
}

File: output/llama_4_maverick/codenet/direct_translation/java/python/s476581255.py

def bubble_sort(a, n):
    flag = True
    count = 0
    while flag:
        flag = False
        for j in range(n - 1, 0, -1):
            if a[j] < a[j - 1]:
                a[j], a[j - 1] = a[j - 1], a[j]
                count += 1
                flag = True
        n -= 1
    return count

n = int(input())
a = list(map(int, input().split()))
count = bubble_sort(a, n)
print(*a)
print(count)

